dev/RadioButtons/RadioButtons.cpp

```diff
@@ -29,6 +29,8 @@ RadioButtons::RadioButtons()
     AccessKeyInvoked({ this, &RadioButtons::OnAccessKeyInvoked });
     GettingFocus({ this, &RadioButtons::OnGettingFocus });
 
+    m_radioButtonsElementFactory = winrt::make_self<RadioButtonsElementFactory>();
```

Comment:
>RadioButtonsElementFactory [](start = 52, length = 26)

Is this needed? you've updated the DP so that the default value is one of these already, can we use that instead of making a new one here?

Reply:
@StephenLPeters I've updated the DP to *not* be a RadioButtonsElementFactory. With this PR, the value of the `ItemTemplate` DP will be used by this `RadioButtonsElementFactory` instance created here as the ContentTemplate of the Content to be wrapped inside a `RadioButton`, thus ensuring that items added to the `RadioButtons` control will *always* be a `RadioButton`.